assimilation_code/modules/assimilation/adaptive_inflate_mod.f90

```diff
@@ -636,65 +678,6 @@ subroutine bayes_cov_inflate(ens_size, inf_type, x_p, sigma_p_2, y_o, sigma_o_2,
 ! Squared Innovation
 dist_2 = (y_o - x_p)**2
    
-! this block of code no longer being used.  it's here for historical purposes.
```

Comment:
is removing this ok with jeff?   it certainly could be moved to the end of this file in a separate comment block with some explanation about when it was done.